src/pages/logosValidator/dashboardDefinition.ts

```diff
@@ -96,8 +96,256 @@ export const LOGOS = {
     link: "https://www.inao.gouv.fr/Les-signes-officiels-de-la-qualite-et-de-l-origine-SIQO/Label-Rouge",
     logo: "https://static.openfoodfacts.org/images/lang/fr/labels/label-rouge.90x90.svg",
   },
+  // Various origins.
+  "en:fsc-mix": {
+    tag: "en:fsc-mix",
+    label: "FSC Mix",
+    logo: "https://static.openfoodfacts.org/images/lang/en/labels/fsc-mix.90x90.svg",
+  },
+  "en:fsc": {
+    tag: "en:fsc",
+    label: "FSC",
+    logo: "https://static.openfoodfacts.org/images/lang/en/labels/fsc.90x90.svg",
+  },
+  "en:pefc": {
+    tag: "en:pefc",
+    label: "PEFC",
+    logo: "https://static.openfoodfacts.org/images/lang/en/labels/pefc.90x90.svg",
+  },
+  "en:green-dot": {
+    tag: "en:green-dot",
+    label: "Green Dot (means the manufacturer paid a contribution to CITEO)",
+    logo: "https://static.openfoodfacts.org/images/lang/en/labels/green-dot.90x90.svg",
+  },
+  "fr:triman": {
+    tag: "fr:triman",
+    label: "Triman (means it's actually recyclable)",
+    logo: "https://static.openfoodfacts.org/images/lang/fr/labels/triman.90x90.svg",
+  },
+
+  "en:dolphin-safe": {
+    tag: "en:dolphin-safe",
+    label: "Dolphin Safe",
+    logo: "https://static.openfoodfacts.org/images/lang/en/labels/dolphin-safe.90x90.svg",
+  },
+  "en:canada-organic": {
+    tag: "en:canada-organic",
+    label: "Canada Organic",
+    logo: "https://static.openfoodfacts.org/images/lang/en/labels/canada-organic.90x90.svg",
+  },
+  "de:ohne-gentechnik": {
+    tag: "de:ohne-gentechnik",
+    label: "Ohne Gentechnik",
+    logo: "https://static.openfoodfacts.org/images/lang/fr/labels/de:ohne-gentechnik.90x90.svg",
+  },
+  "en:made-in-france": {
+    tag: "en:made-in-france",
+    label: "Fabriqué en France",
+    logo: "https://static.openfoodfacts.org/images/lang/en/labels/made-in-france.90x90.svg",
```

Comment:
does not exist

Reply:
Better to add a comment in code such that we can track them

Reply:
done